engine/engine/content/builtins/manifests/web/engine_template.html

```diff
@@ -204,6 +204,7 @@
 	</script>
 
 	<script id='engine-start' type='text/javascript'>
+		EngineLoader.stream_wasm = "{{html5.wasm_streaming}}" === "true";
```

Comment:
Did you check this?

Reply:
If it works you mean? Yes, I've tested it.